File: 7. Reverse Integer.js
// 7. Reverse Integer
x = 123
/**
 * @param {number} x
 * @return {number}
 */
var reverse = function(x) {
  let n = parseInt(x.toString().split('').reverse().join(''));
  if (n > 2147483647) {
      return 0;
  }
  return x < 0 ? -n : n;
}
console.log(reverse(x)); 

File: 7. Reverse Integer.cs
// 7. Reverse Integer
// = 123
//
public class Solution {
    public int Reverse(int x) {
        int n = int.Parse(string.Concat(Math.Abs(x).ToString().Reverse()));
        return n > 2147483647 ? 0 : (x < 0 ? -n : n);
    }
}
